pkg/apis/serving/v1beta1/predictor.go

```diff
@@ -81,6 +81,25 @@ type PredictorExtensionSpec struct {
 	// Each framework will have different defaults that are populated in the underlying container spec.
 	// +optional
 	v1.Container `json:",inline"`
+	// Storage Spec for model location
+	// +optional
+	Storage *StorageSpec `json:"storage,omitempty"`
+}
+
+type StorageSpec struct {
```

Comment:
Is there any reason why storage type is not on the spec?

Reply:
see https://github.com/kserve/kserve/pull/1899#discussion_r787158486